ansible/roles/debops.postldap/defaults/main.yml

```diff
@@ -0,0 +1,627 @@
+                                         )
+                                         (mailEnabled=TRUE)
+                                         (|
+                                           (authorizedService=postfix)
+                                           (authorizedService=mail)
+                                           (authorizedService=\*)
+                                         )'
+
+                                                                   # ]]]
+# .. envvar:: postldap__ldap_domain_filter [[[
+#
+# The LDAP filter used to look up user accounts in the directory.
+postldap__ldap_domain_filter: '(&
+                                            (ObjectClass=dNSDomain)
+                                            (dc=%s)
+                                           )'
+
+                                                                   # ]]]
+# .. envvar:: postldap__ldap_quota_attribute [[[
+#
+# Set the LDAP attribute value to be read by Postfix in order to get the user quota.
+postldap__ldap_quota_attribute: 'mailQuota'
+                                                                   # ]]]
+# .. envvar:: postldap__ldap_quota_default [[[
+#
+# User default LDAP quota. Use human-readable values, e.g. "2 GB".
+postldap__ldap_quota_default: '10 GB'
+                                                                   # ]]]
+# .. envvar:: postldap__ldap_default_config [[[
+#
+# The LDAP configuration options defined by default.
+postldap__ldap_default_config: []
+
+                                                                   # ]]]
+# .. envvar:: postldap__ldap_config [[[
+#
+# List of custom LDAP configuration options defined for all hosts
+# in the Ansible inventory.
+postldap__ldap_config: []
+                                                                   # ]]]
+
+# .. envvar:: postldap__group_ldap_config [[[
+#
+# List of custom LDAP configuration options defined on hosts in
+# a specific Ansible inventory group.
+postldap__group_ldap_config: []
+                                                                   # ]]]
+
+# .. envvar:: postldap__host_ldap_config [[[
+#
+# List of custom LDAP configuration options defined on specific
+# hosts in the Ansible inventory.
+postldap__host_ldap_config: []
+                                                                   # ]]]
+
+# .. envvar:: postldap__ldap_combined_config [[[
+#
+# The variable that combines default and user LDAP configuration and is used in
+# the role tasks and templates.
+postldap__ldap_combined_config: '{{ postldap__ldap_default_config
+                                              + postldap__ldap_config
+                                              + postldap__group_ldap_config
+                                              + postldap__host_ldap_config }}'
+                                                                   # ]]]
+# .. envvar:: postldap__ldap__dependent_tasks [[[
```

Comment:
The dependent variables should be put in their own section at the end of the `defaults/main.yml` file. I would also consider moving other dependent variables here for consistency.

Reply:
IMHO: Moving the dependent variables to a section "Configuration for other Ansible roles" at the end, breaks the logic behind the organisation of the variables. 
I tested your proposition by moving all dependent variables to the end, but then configuring the role while reading the docs is weird. Sections are split, it seems incoherent to me.

Although I acknowledge my proposal has a different structure than other roles, this role modifies postfix heavily, thus building the configuration docs in "chronological" order (what needs to be done first) helps for understanding.
But if you want it changed, its your call.

Reply:
I see. Readability and understanding of complex roles is important, probably even more so than the rules about variable order, and some roles can have exceptions to the rules. If you think that the current order better explains how the role works, let's keep it. :-)